pandas_gbq/gbq.py

```diff
@@ -1091,6 +1096,12 @@ def _generate_bq_schema(df, default_type="STRING"):
     return schema.generate_bq_schema(df, default_type=default_type)
 
 
+def _update_bq_schema(schema_old, schema_new):
+    from pandas_gbq import schema
+
+    return schema.update_schema(schema_old, schema_new)
```

Comment:
Do we need this function? Should we import from `schema` directly? Or there's a circular import?

Reply:
I was just following the pattern used for the only other `schema` function, which is imported the same way in `gbq._generate_bq_schema`. As far as I'm concerned it's fine to get rid of it. Let me know and I'll make the change.

Reply:
I'll defer to Chesterton's fence; we can clean up later if @tswast knows